resources/assets/lib/profile-page/parse-event.tsx

```diff
@@ -60,6 +64,7 @@ export default function parseEvent(event: EventJson, modifiers: Modifiers): { ba
 
     case 'beatmapsetUpdate':
       return {
+        badge: <span className='fas fa-sync-alt profile-extra-entries__icon--green' />,
```

Comment:
```suggestion
        badge: <span className='fas fa-sync-alt profile-extra-entries__icon--green' />,
        iconModifiers: 'green',
```

And then adjust the render function to use the modifiers and apply it correctly to the `__icon` element.